cmd/installer/pkg/install/errata.go

```diff
@@ -61,6 +61,11 @@ func errataArm64ZBoot() {
 		return
 	}
 
+	if bytes.Contains(contents, []byte("CONFIG_EFI_ZBOOT=y")) {
```

Comment:
kernel config changed upstream, so now we check both `CONFIG_EFI_ZBOOT` and `CONFIG_ARM64_ZBOOT`

Reply:
nice catch!

and I think we can probably remove this altogether if we don't support upgrading from a version that first introduced ZBOOT?

Reply:
double checked, this errata is only for Talos < 1.8.0, while minimum host to upgrade to 1.10.0 is Talos 1.8.0, so we can drop this whole errata function altogether

Reply:
dropped :+1: